StaticAnalyzer/views/android/static_analyzer.py

```diff
@@ -508,3 +524,24 @@ def gen_downloads(app_dir, md5, icon_path=''):
 
     except:
         PrintException("Generating Downloads")
+
+
+def get_app_name(app_path, app_dir, tools_dir, is_apk):
+    """ get app name """
+    
+    data = ""
+    if is_apk:
+        output_dir = os.path.join(app_dir, "apktool_out")
+        strings_file = os.path.join(output_dir, "res/values/strings.xml")
+    else:
+        strings_file = os.path.join(app_dir, 'app/src/main/res/values/strings.xml')
```

Comment:
What if it's an old Eclipse style android project?
`app/src/main/res/values/` directory exists?

Reply:
I don't know, I don't have many test samples.